mne/io/eeglab/eeglab.py

```diff
@@ -0,0 +1,140 @@
+# Author: Mainak Jas <mainak.jas@telecom-paristech.fr>
+#
+# License: BSD (3-clause)
+
+import os.path as op
+import numpy as np
+from scipy import io
+
+from ...utils import logger
+from ..meas_info import create_info
+from ..base import _BaseRaw
+from ..constants import FIFF
+
+
+def _topo_to_sph(theta, radius):
+    """Convert 2D topo coordinates to spherical.
+    """
+    sph_phi = (0.5 - radius) * 180
+    sph_theta = -theta
+    return sph_phi, sph_theta
+
+
+def _sph_to_cart(azimuth, elevation, r):
+    """Convert spherical to cartesian coordinates.
+    """
+    x = r * np.cos(elevation) * np.cos(azimuth)
+    y = r * np.cos(elevation) * np.sin(azimuth)
+    z = r * np.sin(elevation)
+    return x, y, z
+
+
+def _get_info(eeg, eog, ch_fname):
+    """Get measurement info.
+    """
+    if not ch_fname.endswith('.locs'):
+        msg = """Currently, only the .locs file format is supported.
+              Please contact mne-python developers for more information"""
+        raise NotImplementedError(msg)
+
+    ch_names = np.loadtxt(ch_fname, dtype='S4', usecols=[3]).tolist()
+    info = create_info(sfreq=eeg.srate, ch_names=ch_names)
+
+    # load channel locations
+    dtype = {'names': ('angle', 'radius'), 'formats': ('f4', 'f4')}
+    angle, radius = np.loadtxt(ch_fname, dtype=dtype, usecols=[1, 2],
+                               unpack=True)
+    sph_phi, sph_theta = _topo_to_sph(angle, radius)
+    sph_radius = np.ones((eeg.nbchan, ))
+    x, y, z = _sph_to_cart(sph_theta / 180 * np.pi,
+                           sph_phi / 180 * np.pi, sph_radius)
+    if eog is None:
+        eog = [idx for idx, ch in enumerate(ch_names) if ch.startswith('EOG')]
```

Comment:
Very common names are 'VEOG' and 'HEOG', so 'in', not 'startswith'. Also, I would much prefer if I could read in all channels as regular channels, even if some have names including 'EOG' (I never treat EOG files as special files).